build.gradle

```diff
@@ -52,6 +53,10 @@ dependencies {
     implementation 'org.springframework.boot:spring-boot-starter-websocket'
 
     implementation 'org.springframework.boot:spring-boot-starter-validation'
+    implementation 'org.springframework.boot:spring-boot-starter-security:2.7.0'
```

Comment:
When I removed this security module all tests pass again. Looking around
https://stackoverflow.com/questions/46745923/why-cant-i-import-withmockuser-to-my-test suggests importing security-test
https://docs.spring.io/spring-security/site/docs/4.2.x/reference/html/test-method.html
https://www.baeldung.com/spring-security-integration-tests

I think we need these 2, but couldn't get them to work
testImplementation('org.springframework.boot:spring-boot-starter-test')
testImplementation('org.springframework.security:spring-security-test')

Reply:
Thank you for your work! 👍🏻